Optional-public-events-dataset-requirements.md

```diff
@@ -0,0 +1,23 @@
+EventAn optional public meetings dataset is included in case there is more than one channel or opportunity for public engagement with regards to a particular consultation project. This can include past or scheduled events.
+
+### Required
+
+Field                | DataType          | Description
+---------------------|-------------------|------------
+ConsultationId       | TEXT              | Provides data link back to core applications information by application ID.
+PublicEventStartDate | TEXT (YYYY-MM-DD) | The date the public event was or will be held. If the event will span multiple days, include a value in the `PublicEventEndDate` field.
+PublicEventTitle     | TEXT              | A short-form description of the nature of the public event.
+
+### Optional
+
+Field                  | DataType          | Description
+-----------------------|-------------------|------------
+PublicEventDescription | TEXT              | A detailed description of the nature of the public event.
+PublicEventAddress1    | TEXT              | The street address of the public event, if applicable, or the first line of the address for multi-line addresses.
+PublicEventAddress2    | TEXT              | The second line of the address for multi-line addresses.
+PublicEventCity        | TEXT              | The city of the address.
+PublicEventState       | TEXT              | The province or state of the address.
```

Comment:
Use the more generic terms "locality" and "region" instead of "city" and "state". Schema.org, vCard and many others use those generic terms.